src/containers/Register/Register.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import './Register.scss';
+import { connect } from 'react-redux';
+import { registerUser } from '../../actions';
+
+class Register extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      first_name: '',
+      last_name: '',
+      email: '',
+      username: '',
+      password: ''
+    };
+
+    this.handleFirstNameOnChange = this.handleFirstNameOnChange.bind(this);
+    this.handleLastNameOnChange = this.handleLastNameOnChange.bind(this);
+    this.handleEmailOnChange = this.handleEmailOnChange.bind(this);
+    this.handleUsernameOnChange = this.handleUsernameOnChange.bind(this);
+    this.handlePasswordOnChange = this.handlePasswordOnChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  };
+
+  componentWillMount() {
+    document.addEventListener('mousedown', this.handleClick, false);
+    document.querySelector('.columnContainer').classList.add('blur');
```

Comment:
commented
Instead of using document, try to directly add the class in the render function using className and the ternary operator